README.md

```diff
@@ -128,6 +164,8 @@ The commands there are especially useful for Linux users coming to FreeBSD.
   - [Workaround](https://github.com/pgj/freebsd-wifibox) by using a Linux virtual machine
 - AX210 Bluetooth not supported yet
   - https://reviews.freebsd.org/D44861
+- RZ717 not supported yet
```

Comment:
Should sort this one after RZ616, or probably makes sense to just have `RZ616/RZ717 not yet supported` (assuming they are going to be the same driver)?

Reply:
> assuming they are going to be the same driver

may, may not
On Linux it's a separate driver but sharing a lot of code, i think.
We can update when we know more.